chain-signatures/node/redis.conf

```diff
@@ -0,0 +1,8 @@
+# Enable Append Only File (AOF) persistence
+appendonly yes
+
+# Set the appendfsync policy to flush the AOF every second
+appendfsync everysec
```

Comment:
We can afford to persist every call, but that will increase latency 4x. 1s should be a good compromise, we can lose a few presignatures now.